operator/api/v1alpha1/observatorium_types.go

```diff
@@ -55,14 +55,14 @@ type ObjectStorageConfig struct {
 
 type ThanosReceiveControllerSpec struct {
 	// Receive Controller image
-	Image string `json:"image"`
+	Image string `json:"image,omitempty"`
```

Comment:
I would argue that _if_ `image` is set, we want to force users to also set version. So they understand that they need to appropriately set `version` according to what's in the `image` they specify.

Reply:
Well, this is not especially related to this patch, but I had always doubts about the `version` field and for is the purpose of it.
I am not sure how to address your comment actually.

Reply:
I'd be curious if we could have conditional validation in the openapischema? Not blocking this I think, but we should have an issue to track this.

Reply:
I open issue #288 for follow up